docs/howto/writing-migrations.txt

```diff
@@ -184,6 +184,45 @@ the respective field according to your needs.
   migration is running. Objects created after the ``AddField`` and before
   ``RunPython`` will have their original ``uuid``’s overwritten.
 
+.. _non-atomic-migrations:
+
+Non-atomic migrations
+~~~~~~~~~~~~~~~~~~~~~
+
+On databases that support DDL transactions (SQLite and PostgreSQL), migrations
+will be run inside a transaction. For some use-cases, like performing data
+migrations on large tables, you may want to exempt a migration from being run
+in a transaction by setting the ``atomic`` class attribute to ``False``::
+
+    from django.db import migrations
+
+    class Migration(migrations.Migration):
+
+        atomic = False
+
+``RunPython`` operations will still be run inside a transaction unless
```

Comment:
Could you add a note about this in the RunPython documentation in `docs/ref/migration-operations.txt`? We still have the old signature there : `.. class:: RunPython(code, reverse_code=None, atomic=True, hints=None)`